tests/scenes/biosphere/test_leaf_cloud.py

```diff
@@ -80,7 +80,7 @@ def test_leaf_cloud_orientations(rng):
     """Unit tests for :func:`_leaf_cloud_orientations`."""
     orientations = _leaf_cloud_orientations(1, 1, 1, rng)
     assert orientations.shape == (1, 3)
-    assert np.allclose(orientations, [0.36553973, 0.85464537, 0.36873025])
+    assert np.allclose(orientations, [0.2323117, -0.75741865, 0.61020346])
```

Comment:
Same here: this just tests for regression of the RNG + function block, not the function itself. We can discuss how to improve that if you wish.

Reply:
I can implement the distribution from goel and strebel and do a statistical comparison of my two implementations, if that makes sense... But we don't really have an external reference for these statistical results which would exclude programming mistakes I could be making... :/

Reply:
I think the best approach is to hard-code the samples you want to use for testing—just like what we do when we test sampling code in Mitsuba. This way you don't have any risk of regression due to the RNG being updated.

Reply:
As for testing the distribution itself, it's a broader-scope upgrade: Doing this without designing a proper statistical test framework would be wasteful IMO. So I'd advise against for now.